docs/meetings/web/2021-11-03.md

```diff
@@ -0,0 +1,19 @@
+# Web meeting Nov 03 2021
+
+**Present:** Aapeli, Chad, Lucas, Ricardo, William
+
+## Discussion
+
+- Translations: all (?) strings now exist in json files, but app is still using the constants. Need to move across
+
+- NextJS in progress
+  - May need to downgrade to v11 for a while if I can't get tests working on 12
```

Comment:
What's different in v11 compared to v12?

Reply:
12 uses a new rust transpiler and I'm having trouble getting the tests to run (I think there is a way to use Babel though so I'll try that first).